app/javascript/components/patient/vaccines/VaccineModal.js

```diff
@@ -165,6 +165,10 @@ class VaccineModal extends React.Component {
                   value={this.state.dose_number ? { label: this.state.dose_number, value: this.state.dose_number } : ''}
                   options={this.getDropdownOptions(this.props.dose_number_options)}
                   onChange={this.handleDoseNumberChange}
+                  theme={theme => ({
```

Comment:
When this modal was created, the `theme` styling was left off of this `<Select>`.

Reply:
Nice catch!